results/schema/results.sql

```diff
@@ -1,4 +1,5 @@
 CREATE TABLE results (
-	name varchar(256) PRIMARY KEY,
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
We should avoid using incrementing integers for primary keys - this caps the size of the database based on the size of the int and this can limit our write throughput for distributed workloads if all writes are required to go to the tail.

I'd recommend using a UUID.

Reply:
This also has the consequence of making CreateResult requests non-unique (e.g. I can call CreateResult with the same name multiple times).

If we're going to change the key, we should probably tackle this in another PR to make sure we tackle the edge cases. Otherwise, can we leave the key as is for now?

Reply:
Okay! But we need an ordered field to support pagination, right? I tried to add a timestamp field, but then I thought that there might be multiple records added to the database in the same time, in such case, timestamp is invalid, therefore I turned to an auto-increment integer primary key. Is there any common way to implement this?